x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -72,6 +74,32 @@ function throwIfUpdateOwner(requests: UpdateRequestWithOriginalCase[]) {
   }
 }
 
+/**
+ * Throws an error if any of the requests attempt to create a number of user actions that would put
+ * it's case over the limit.
+ */
+async function throwIfMaxUserActionsReached({
+  userActionsDict,
+  userActionService,
+}: {
+  userActionsDict: UserActionsDict;
+  userActionService: CaseUserActionService;
+}) {
+  const result = await userActionService.getMultipleCasesUserActionsTotal({
+    caseIds: Object.keys(userActionsDict),
+  });
+
+  result.aggregations?.references.caseUserActions.buckets.forEach(
```

Comment:
For each case check if the result of the aggregation + the user actions to be created is bigger than the max allowed value.

Reply:
Does this mean once reaching maximum limit user can't delete or update the existing comment?